client/signup/step-wrapper/index.jsx

```diff
@@ -71,7 +71,7 @@ class StepWrapper extends Component {
 						flowName={ this.props.flowName }
 						stepName={ this.props.stepName }
 						labelText={ this.props.skipLabelText }
-						cssClass={ !! this.props.skipHeadingText && ' navigation-link--has-skip-heading ' }
+						cssClass={ ! this.props.skipHeadingText && ' navigation-link--has-skip-heading ' }
```

Comment:
This change doesn't seem to have an effect on the theme picking steps, but the logic doesn't seem right here. I feel like `has-skip-heading` would be applied if the heading text was present/non-empty.

But yeah, I can't see what effect this has on our flows. I know jetpack use the skip button more though.

Reply:
https://github.com/Automattic/wp-calypso/pull/36825#discussion_r343641893